### PR TITLE
Remove question about how often do people do coding/design work

### PR DESCRIPTION
**Pull request by Kobzol**
This question was originally suggested [here](https://github.com/rust-lang/surveys/pull/234/files#r1347513327).

While I think that the idea wasn't bad, I don't think that this question currently pulls its weight, so I'd like to remove it to shorten the survey. I'm basing this on the following two reasons:
1) This question only makes sense if we would do some complex cohort analysis and separate the answers of individual people. However, while SurveyHero can do this partially, we don't have access to it, and there's currently no automation (frankly, nor anyone who would have the time/motivation) that would help us with it. And without such analysis, this question is IMO not very useful.
2) Last year, we got the following answers:

In short, ~95 % people answered that they do coding or design work daily or weekly. I think that this shows the structure of our audience, that is probably highly technical. I think that we don't need to repeat the same question next time; I don't expect that it will change a lot.

CC @alice-i-cecile

### CODE REVIEW

**Comment by Kobzol** [comment]
Regarding point 2, we also have the "Do you write or design software in your work?" question later in the survey.

**Comment by alice-i-cecile** [comment]
Agreed, I don't think this is particularly valuable information to gather.

**Review by alice-i-cecile — APPROVED** [review]
